packages/node_modules/@node-red/editor-client/src/js/ui/editor.js

```diff
@@ -426,14 +426,8 @@ RED.editor = (function() {
             }
         });
 
-        var label = "";
-        var configNode = RED.nodes.node(nodeValue);
-
-        if (configNode) {
-            label = RED.utils.getNodeLabel(configNode, configNode.id);
-        }
-
-        input.val(label);
+        // Set the select value
+        select.val(nodeValue);
```

Comment:
```suggestion
        const optionCount = select.find("option").length
        if (optionCount === 1) {
            // We only have the 'add new...' option
            select.val('_ADD_')
        } else {
            select.val(nodeValue);
        }
```

The change as proposed doesn't properly handle the empty list case, where the 'none' option should actually show the 'add new...' option which has a value of `_ADD_`.

Reply:
Not sure, this part is managed by `updateConfigNodeSelect`.

Reply:
Strange, as this option is the only one and is selected, it surprises me that it doesn't work.